schemas/assessment-schema-1.2.json

```
@@ -0,0 +1,150 @@
+                        },
+                        "result": {
+                            "$ref": "#/$defs/action_result"
+                        }
+                    }
+                }
+            }
+        },
+        "format_version": {
+            "description": "Constant value that tells us the format of this file.",
+            "const": "1.1"
+        },
+        "status": {
+            "description": "The highest severity between messages and results from actions.",
+            "type": "string",
+            "enum": [
+                "SUCCESS",
+                "INFO",
+                "WARNING",
+                "SKIP",
+                "OVERRIDABLE",
+                "ERROR"
+            ]
+        }
+    },
+    "required": [
+        "actions",
+        "format_version",
+        "status"
+    ],
+
+    "$defs": {
+        "result_levels": {
+            "description": "The severity of the result",
+            "type": "string",
+            "enum": [
+                "SUCCESS",
+                "SKIP",
+                "OVERRIDABLE",
+                "ERROR"
+            ]
+        },
+        "message_levels": {
+            "description": "The severity of the message",
+            "type": "string",
+            "enum": [
+                "INFO",
+                "WARNING"
+            ]
+        },
+        "base_action_message": {
+            "type": "object",
+            "properties": {
+                "title": {
+                    "description": "Short, one line summary of the message.",
+                    "type": "string"
+                },
+                "description": {
+                    "description": "Longer description of the purpose of this message.",
+                    "type": "string"
+                },
+                "diagnosis": {
+                    "description": "How this message applies to this particular system. For instance, 'This system has convert2rhel-1.0 but convert2hel-2.2 is the latest.'",
+                    "type": "string"
+                },
+                "id": {
+                    "description": "Identifier for this message. The combination of the action_result's id and this message id will be unique.",
+                    "type": "string",
+                    "pattern": "^[A-Z0-9_]+$"
+                },
+                "remediations": {
+                    "description": "Steps the user may take to fix this issue.",
+                    "type": "string"
+                },
+                "variables": {
+                    "description": "Information about this particular system that may be used to template the diagnosis and remediation fields.",
+                    "type": "object",
+                    "patternProperties": {
+                        "^[A-Za-z0-9_]+$": {
+                        }
+                    }
+                }
+            },
+            "required": ["title", "description", "diagnosis", "id", "remediation", "variables"]
```

```suggestion
            "required": ["title", "description", "diagnosis", "id", "remediations", "variables"]
```
I believe this needs to be updated to remediations as well

---

good catch!